desktop/mac/release-notes/index.md

```diff
@@ -33,6 +33,46 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### New
+
+- Released Docker Desktop for Linux.
+- Beta release of Docker Extensions and Extensions SDK.
```

Comment:
Can we add a link to the Extensions doc - https://docs.docker.com/desktop/extensions